exercises/practice/knapsack/.docs/instructions.md

```diff
@@ -26,7 +26,7 @@ Items: [
 
 Knapsack Limit: 10
 
-For the above, the first item has weight 5 and value 10, the second item has weight 4 and value 40, and so on.
+For the above, the first item has a weight of 5 and a value of 10, the second item has a weight of 4 and a value of 40, and so on.
```

Comment:
I think this too is from the original instructions. If it is, we shouldn't update them here.